### PR TITLE
timestamp cleanup

### PR DESCRIPTION
**Pull request by malaffoon**
DWR-329

### CODE REVIEW

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/model/AnyExam.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 public abstract class AnyExam {
     //TODO: add support for accessibility accommodations
-    private Date completedAt;
+    private Instant completedAt;
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/model/AnyExam.java]: This value needs to be a full datetime. I am recommending that all such fields be Instant's.

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java [hunk 2]
```diff
@@ -60,7 +61,7 @@ public long create(final Exam exam) {
                 .addValue("scale_score", exam.getScaleScore())
                 .addValue("scale_score_std_err", exam.getScaleScoreStdErr())
                 .addValue("achievement_level", exam.getAchievementLevel())
-                .addValue("completed_at", exam.getCompletedAt());
+                .addValue("completed_at", Timestamp.from(exam.getCompletedAt()));
```
Comment (malaffoon) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: Putting an Instant into a SQL TIMESTAMP field is easy.

**Comment by bgsavage** [comment]
Does the ImportRepositoryImpl need to be updated as well?

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java (hunk 2 above)
Comment (jtreuting) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: oops, commented before seeing this comment.

File: exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java [hunk 3]
```diff
@@ -73,7 +73,7 @@
             "administration_condition_id = 1 and " +
             "asmt_version = '345' and " +
             "asmt_id = -1 and " +
-            "completed_at = '2007-01-02' and " +
+            "completed_at = convert_tz(timestamp('2007-01-02T14:30:00Z'), '+00:00', @@session.time_zone) and " +
```
Comment (malaffoon) [inline exam-processor/src/test/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryIT.java]: And here is where it gets fun. When doing native mysql client queries you have to convert timezones. Sucks. Probably means Jeff's warning will be an issue.

File: exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java (hunk 2 above)
Comment (jtreuting) [inline exam-processor/src/main/java/org/opentestsystem/rdw/ingest/processor/repository/impl/ExamRepositoryImpl.java]: If it is an issue, then the solution that we found was to do this:

`new Timestamp(exam.getCompletedAt().getMillis())`

**Pull request merged**